spacialgaze-plugins/news.js

```diff
@@ -0,0 +1,86 @@
+/*
+* News System for SpacialGaze
+* Credits: Lord Haji
+*/
+
+'use strict';
+
+let moment = require('moment');
+
+function generateNews() {
+	let newsData , newsDisplay = [];
+	Object.keys(Db('news').object()).forEach (announcement => {
+		newsData = Db('news').get(announcement);
+		newsDisplay.push(`<h4>${announcement}</h4>${newsData[1]}<br /><br />—${SG.nameColor(newsData[0], true)} <small>on ${newsData[2]}</small>`);
+	});
+	return newsDisplay;
+}
+
+function hasSubscribed(user) {
+	if (typeof user === 'object') user = user.userid;
+	if (Db('NewsUnsubscribers').get(toId(user)) === 1) return false;
+	return true;
+}
+
+SG.showNews = function (user) {
+	if (!Users(user)) return false;
+	let newsDisplay = generateNews();
+	if (newsDisplay.length === 0) return false;
+	if (!hasSubscribed(user)) return false;
+	if (newsDisplay.length > 0) {
+		newsDisplay = newsDisplay.join('<hr>');
+		return Users(user).send(`|pm| SpacialGaze News|${Users(user).getIdentity()}|/raw ${newsDisplay}`);
```

Comment:
Might want to use `SG Server` as the name because Spacialgaze News is unregistered (and onemore account for me to track). Im not so sure about witch to use myself, so this is open for discusion.